pgmpy/models/BayesianModel.py

```diff
@@ -382,6 +382,10 @@ def _get_ancestors_of(self, obs_nodes_list):
         if not isinstance(obs_nodes_list, (list, tuple)):
             obs_nodes_list = [obs_nodes_list]
 
+        for i in obs_nodes_list:
```

Comment:
@kris-singh Use a better name for the iterable instead of `i`. Maybe `node` or `var`.